.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -9,6 +9,7 @@ assignees: ''
 
 # User Story
 ### Essential components
+- [ ] Story title is meaningful
```

Comment:
"Meaningful" is ambiguous.  Perhaps just indicate that the title has been changed from the placeholder text.

Reply:
Modify to read "Issue Title placeholder text is modified"

Reply:
"Issue Title is descriptive of the user story"